src/sql/workbench/contrib/notebook/browser/cellViews/linkHandler.directive.ts

```diff
@@ -62,19 +64,22 @@ export class LinkHandlerDirective {
 			if (uri.fragment && uri.fragment.length > 0 && uri.fsPath === this.workbenchFilePath.fsPath) {
 				this.notebookService.navigateTo(this.notebookUri, uri.fragment);
 			} else {
-				let newUri: URI | undefined;
-				let path = dirname(this.notebookUri.path);
-				path = 'file://' + path + '//' + basename(uri.path);
-				try {
-					newUri = URI.parse(path);
-				} catch{
-					//ignore
+				let exists = await this.fileService.exists(uri);
+				if (!exists) {
+					let relPath = relative(this.workbenchFilePath.fsPath, uri.fsPath);
```

Comment:
Why the workbench file path here and not the notebook path?

Reply:
When an anchor link is used, the path from the target it's modified and instead of just showing the relative path, it uses the absolute path of the workbenchFile plus the original relative path. 
That's why I get the difference between these two paths and then I create the absolute path using the notebook path and the relative path.

Reply:
I can get the relative path from target.attributes.href.value, instead of using target['href'], but I'm not sure if this is the right way.

Reply:
Ok, makes sense.